faucet-ui/src/containers/Success.tsx

```diff
@@ -18,7 +18,7 @@ export default (props: any) => {
   return (
     <Box gap="small" align="center">
       <Text textAlign='start' weight="bold" color="text" size='xlarge'> Congratulations! </Text>
-      <Text textAlign='start' weight="bold" color="text" size='xlarge'> You just got 50,000 CKB Testnet tokens! </Text>
+      <Text textAlign='start' weight="bold" color="text" size='xlarge'> You just got 500 CKB Testnet tokens! </Text>
```

Comment:
I believe this should not be hard-coded. Instead, use the constant or configuration as the backend.

This way when we change the amount in the future the UI/text copy still works.

Reply:
Here we will use the environment parameters